content/en/docs/tasks/debug/debug-cluster/monitor-node-health.md

```diff
@@ -100,54 +97,58 @@ This approach only applies to a Node Problem Detector started with `kubectl`.
 Overwriting a configuration is not supported if a Node Problem Detector runs as a cluster Addon.
 The Addon manager does not support `ConfigMap`.
 
-## Kernel Monitor
+## Problem Daemon
+
+A problem daemon is a sub-daemon of the Node Problem Detector. It monitors specific kinds of node
+problems and reports them to the Node Problem Detector.
+There are several types of supported problem daemons.
 
-*Kernel Monitor* is a system log monitor daemon supported in the Node Problem Detector.
-Kernel monitor watches the kernel log and detects known kernel issues following predefined rules.
+- A `SystemLogMonitor` type of daemon monitors the system logs and reports problems and metrics
+  according to predefined rules. You can customize the configurations for different log sources
+  such as [filelog](https://github.com/kubernetes/node-problem-detector/blob/master/config/kernel-monitor-filelog.json),
+  [kmsg](https://github.com/kubernetes/node-problem-detector/blob/master/config/kernel-monitor.json),
+  [kernel](https://github.com/kubernetes/node-problem-detector/blob/master/config/kernel-monitor-counter.json),
+  [abrt](https://github.com/kubernetes/node-problem-detector/blob/master/config/abrt-adaptor.json),
+  and [systemd](https://github.com/kubernetes/node-problem-detector/blob/master/config/systemd-monitor-counter.json).
 
-The Kernel Monitor matches kernel issues according to a set of predefined rule list in
-[`config/kernel-monitor.json`](https://github.com/kubernetes/node-problem-detector/blob/v0.1/config/kernel-monitor.json). The rule list is extensible. You can expand the rule list by overwriting the
-configuration.
+- A `SystemStatsMonitor` type of daemon collects various health-related system stats as metrics.
+  You can customize its behavior by updating its
+  [configuration file](https://github.com/kubernetes/node-problem-detector/blob/master/config/system-stats-monitor.json).
 
-### Add new NodeConditions
+- A `CustomPluginMonitor` type of daemon invokes and checks various node problems by running
+  user-defined scripts. You can use different custom plugin monitors to monitor different
+  problems and customize the daemon behavior by updating the
+  [configuration file](https://github.com/kubernetes/node-problem-detector/blob/master/config/custom-plugin-monitor.json).
```

Comment:
I might link to a specific commit, which protects against https://github.com/kubernetes/node-problem-detector/ getting reorganized.

Reply:
Pinned to 0.8.12 tag now.